recipe/meta.yaml

```
@@ -1,58 +1,70 @@
 {% set name = "hydra-core" %}
-{% set version = "1.3.1" %}
+{% set version = "1.3.2" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://github.com/facebookresearch/hydra/archive/v{{ version }}.tar.gz
-  sha256: 9ba1efe09893853e883498fcbeec17f2a77e1ba551587d6d0604077e02de1602
+  - url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+    sha256: 8a878ed67216997c3e9d88a8e72e7b4767e81af37afb4ea3334b269a4390a824
+    patches:
+    # do not forget to update the version literal in the antlr-*-complete.jar string inside the
+    # patch to correctly unvendor antlr and use the antlr package from conda
+      - 0000-unvendor_antlr.patch
+    folder: pypi-upstream
+  - url: https://github.com/facebookresearch/hydra/archive/refs/tags/v1.3.2.tar.gz
+    sha256: 3f0f14305e9c6ae4b1dbcba344fe09dd0fc79c0a0656470c249f4f7ca46bf471
+    folder: test-assets
 
 build:
-  noarch: python
   number: 0
   script:
-    - {{ PYTHON }} setup.py antlr
-    - {{ PYTHON }} -m pip install . -vv
+    - {{ PYTHON }} -m pip install $SRC_DIR/pypi-upstream --no-deps --no-build-isolation -vv
 
 requirements:
   build:
-    - antlr ==4.9.*
+    - patch  # [not win]
+    - m2-patch  # [win]
   host:
-    - python >=3.6
+    - antlr 4.9.3
+    - python
     - pip
+    - openjdk
   run:
-    - python >=3.6
+    - python
     - omegaconf >=2.2,<2.4
-    - antlr-python-runtime ==4.9.*
+    # minor versions are introducing breaking changes (another version was tried, e.g. 4.13.* and implementation
+    # turned out to be incompatible)
+    - antlr4-python3-runtime >=4.9,<4.10
```

```suggestion
    - antlr4-python3-runtime >=4.9,<4.10.0a0
```

or you could leave:

```suggestion
    - antlr4-python3-runtime ==4.9.*
```
https://github.com/facebookresearch/hydra/blob/7ef3acab78b46bf82370e7931f748aef5fd80e5f/requirements/requirements.txt#L2